src/components/include/cos_kernel_api.h

```diff
@@ -107,5 +107,63 @@ int cos_hw_detach(hwcap_t hwc, hwid_t hwid);
 void *cos_hw_map(struct cos_compinfo *ci, hwcap_t hwc, paddr_t pa, unsigned int len);
 int cos_hw_cycles_per_usec(hwcap_t hwc);
 
+/* Default API */
```

Comment:
Might as well document what "default" means.  Otherwise the comment isn't really that useful.